### PR TITLE
chore(aws-load-balancer-controller): update all controllerConfig.featureGates samples default values

### PR DESCRIPTION
**Pull request by kahirokunn**
### Issue

I looked at the default values in v2.5 and they were almost valid, so I adjusted them to the latest state
https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.5/deploy/configurations/#feature-gates

### Checklist
- [ ] Added tests that cover your change (if possible)
- [x] Added/modified documentation as required (such as the `README.md`, or the `docs` directory)
- [x] Manually tested
- [x] Made sure the title of the PR is a good description that can go into the release notes

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [x] Refactored something and made the world a better place :star2:

### CODE REVIEW

**Comment by kahirokunn** [comment]
@kishorj I have reviewed and recorrected the most recent documents.

File: helm/aws-load-balancer-controller/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: aws-load-balancer-controller
 description: AWS Load Balancer Controller Helm chart for Kubernetes
-version: 1.5.1
+version: 1.5.2
```
Comment (kishorj) [inline helm/aws-load-balancer-controller/Chart.yaml:4]: lets keep the chart version at 1.5.1, we will update it when publishing the chart.

Comment (kahirokunn) [reply helm/aws-load-balancer-controller/Chart.yaml]: done

**Review by kishorj — APPROVED** [review]

**Pull request merged**